test/prop_counter_b.erl

```diff
@@ -0,0 +1,102 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright <2013-2018> <
+%%  Technische Universität Kaiserslautern, Germany
+%%  Université Pierre et Marie Curie / Sorbonne-Université, France
+%%  Universidade NOVA de Lisboa, Portugal
+%%  Université catholique de Louvain (UCL), Belgique
+%%  INESC TEC, Portugal
+%% >
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either expressed or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% List of the contributors to the development of Antidote: see AUTHORS file.
+%% Description and complete License: see LICENSE file.
+%% -------------------------------------------------------------------
+
+-module(prop_counter_b).
+
+-define(PROPER_NO_TRANS, true).
+-include_lib("proper/include/proper.hrl").
+
+%% API
+-export([prop_is_operation_test/0, prop_is_not_operation_test/0, prop_increment_decrement_test/0]).
+
+
+%%%%%%%%%%%%%%%%%%
+%%% Properties %%%
+%%%%%%%%%%%%%%%%%%
+
+prop_is_operation_test() ->
+    ?FORALL(Operation, valid_operations(),
+        begin
+            antidote_crdt_counter_b:is_operation(Operation)
+        end).
+
+prop_is_not_operation_test() ->
+    ?FORALL(Operation, invalid_negative_operations(),
+        begin
+            not antidote_crdt_counter_b:is_operation(Operation)
+        end).
+
+prop_increment_decrement_test() ->
```

Comment:
Comments would be helpful here to understand the general structure of the test.

Reply:
Added some comments